siteimprove/admin/js/siteimprove.js

```diff
@@ -39,24 +39,36 @@
       this.common();
     },
     common: function (url) {
-      var _si = window._si || [];  
-
-      var getDomCallback = async function () {
-      	var pageWindow = window.open(
-      		url,
-      		"Page Preview",
-      		"width=400,height=500"
-      	);
-      	var promise = new Promise(function (resolve, reject) {
-      		pageWindow.addEventListener(
+      const _si = window._si || [];  
+    
+      const getDomCallback = async function () {
+        const newDiv = document.createElement("div");
+        newDiv.setAttribute("id","div_iframe"); 
+        document.body.appendChild(newDiv);
+        let	iframeString = "<iframe id='domIframe' src="+ url.concat("&si_preview=1") +" style='height:100vh; width:100%'></iframe>";
```

Comment:
Why is there a "tab" character after `let`?

Reply:
Interesting... VScode did't showed me that way.
But I'll fix it